### ISSUE TITLE
Preferences blank (Firefox)

### ISSUE THREAD

**Issue by sammcj** [foyer-work/ooogle-it#36]
Looks like the preferences are blank:

![image](https://user-images.githubusercontent.com/862951/215590658-734bc5b8-e5f2-4055-8e88-1c49cf399919.png)

FF 110
macOS 13.x

**Comment by siddsax** [comment]
Yup, thats coz of a bug of sorts in firefox. basically to have users be able to come to this page for changing the shortcut, we need to have the options/preference page there. Even if we don't have any preference in app.
